modules/openapi-generator/src/test/java/org/openapitools/codegen/python/PythonClientCodegenTest.java

```diff
@@ -54,4 +59,37 @@ public void testAdditionalPropertiesPutForConfigValues() throws Exception {
         Assert.assertEquals(codegen.isHideGenerationTimestamp(), false);
     }
 
+    @Test(description = "test regex patterns")
+    public void testRegularExpressionOpenAPISchemaVersion3() {
+        final OpenAPI openAPI = new OpenAPIParser()
+                .readLocation("src/test/resources/3_0/issue_1517.yaml", null, new ParseOptions()).getOpenAPI();
+        final PythonClientCodegen codegen = new PythonClientCodegen();
+        final String path = "/ping";
+        final Operation p = openAPI.getPaths().get(path).getGet();
+        final CodegenOperation op = codegen.fromOperation(path, "get", p, openAPI.getComponents().getSchemas());
+        // pattern_no_forward_slashes '^pattern$'
+        Assert.assertEquals(op.allParams.get(0).pattern, "/^pattern$/");
+        // pattern_two_slashes '/^pattern$/'
+        Assert.assertEquals(op.allParams.get(1).pattern, "/^pattern$/");
+        // pattern_dont_escape_backslash '/^pattern\d{3}$/'
+        Assert.assertEquals(op.allParams.get(2).pattern, "/^pattern\\d{3}$/");
+        // pattern_dont_escape_escaped_forward_slash '/^pattern\/\d{3}$/'
+        Assert.assertEquals(op.allParams.get(3).pattern, "/^pattern\\/\\d{3}$/");
+        // pattern_escape_unescaped_forward_slash '^pattern/\d{3}$'
+        Assert.assertEquals(op.allParams.get(4).pattern, "/^pattern\\/\\d{3}$/");
+    }
+
+    @Test(
+            description = "test regex patterns with modifiers",
+            expectedExceptions = {IllegalArgumentException.class},
+            expectedExceptionsMessageRegExp = "Python client does not currently support regular expression modifiers."
+    )
+    public void testRegularExpressionModifiersOpenAPISchemaVersion3() {
+        final OpenAPI openAPI = new OpenAPIParser()
+                .readLocation("src/test/resources/3_0/issue_1517.yaml", null, new ParseOptions()).getOpenAPI();
+        final PythonClientCodegen codegen = new PythonClientCodegen();
+        final String path = "/modifiers";
+        final Operation p = openAPI.getPaths().get(path).getGet();
+        final CodegenOperation op = codegen.fromOperation(path, "get", p, openAPI.getComponents().getSchemas());
```

Comment:
Yup, it does. If you look at the `@Test` annotation it declares that the test is expected to throw an IllegalArgumentException with the error message "Python client does not currently support regular expression modifiers."  If you comment those two lines out and try running the test, it will fail with the aforementioned exception/message.

Reply:
I see it now, thanks.